Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -5160,7 +5160,7 @@ size_t Document::broadcast_active_resize_observations()
     auto shallowest_target_depth = NumericLimits<size_t>::max();
 
     // 2. For each observer in document.[[resizeObservers]] run these steps:
-    for (auto const& observer : m_resize_observers) {
+    for (auto resize_observers = m_resize_observers; auto const& observer : resize_observers) {
```

Comment:
This is not enough; the copy here will be a `Vector<GC::Ref<ResizeObserver>>` but nothing visits those.
We need to use either `Vector<GC::Root<ResizeObserver>>` or `MarkedVector<GC::Ref<ResizeObserver>>`